runtime/vm/JFRConstantPoolTypes.hpp

```diff
@@ -19,10 +19,11 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
  *******************************************************************************/
-#include <cstring>
 #if !defined(JFRCONSTANTPOOLTYPES_HPP_)
 #define JFRCONSTANTPOOLTYPES_HPP_
 
+#include <cstring>
```

Comment:
Please move this *after* the openj9 and omr includes, as requested; then squash.